app/controllers/admin/events_controller.rb

```diff
@@ -5,6 +5,9 @@ def index
     if params[:type].blank?
       @data = events.group_by_type_in_range(start_date, end_date)
       @columns = Ahoy::Event.action_types(action_page)
+      if action_page.present? && action_page.congress_message_campaign.present?
```

Comment:
It might be better to check `enable_congress_message?` instead, since the form for action pages will create tool associations even when they're not used.